File: Zestawy Zadań/C++/1.1 - Operatory wyjscia-wejscia/Zestaw 2 zadania/1.6.cpp
//Napisz program, który potrafi odszukać miejsce zerowe dowolnej prostej zadanej w postaci
//kierunkowej y= ax + b (przy założeniu, że a≠ 0). Program :
//a. Pobiera od użytkownika współczynniki a i b
//b. W kolejnym wierszu wypisuje znalezione miejsce zerowe
//c. W kolejnym wierszu wykonuje pauzę systemową

#define _USE_MATH_DEFINES
#include <iostream>
#include <iomanip>
#include <math.h>

using namespace std;

int main()
{
    float a, b, zerowe = 0, pauza;
    cin >> a >> b;
    if (a)
    {
        zerowe = -b / a;
        cout << "Znaleziono miejsce zerowe. Wspolrzedne: (" << zerowe << ",0)" << endl;
        cin >> pauza;
    }
}

File: Zestawy Zadań/Python/1.1 - Operatory wyjścia - wejścia/Zestaw 2 zadania/1.6.py
#Napisz program, który potrafi odszukać miejsce zerowe dowolnej prostej zadanej w postaci
#kierunkowej y= ax + b (przy założeniu, że a≠ 0). Program :
#a. Pobiera od użytkownika współczynniki a i b
#b. W kolejnym wierszu wypisuje znalezione miejsce zerowe
#c. W kolejnym wierszu wykonuje pauzę systemową

import time

a = float(input("Podaj a: "))
b = float(input("Podaj b: "))

if a != 0:
    zerowe = -b/a
    print("Współrzędne zerowego: (", zerowe, ",0)")
    time.sleep(1)
    input()